FLIR/conservator/run_tasks_plugins/conservator_cli.py

```diff
@@ -0,0 +1,11 @@
+from FLIR.conservator.cli import main
+
+
+def conservator_cli(cmd_class, cmd_name, args=(), log="INFO", task=None):
```

Comment:
It looks like `log` is never used

Reply:
Since not all commands are exactly 2 subcommands deep (such as `conservator whoami`), I'd suggest switching to only a single `command` and list of `args`.

Reply:
Right on both counts